src/mcts/node.h

```diff
@@ -336,8 +337,10 @@ class EdgeAndNode {
   Node* node() const { return node_; }
 
   // Proxy functions for easier access to node/edge.
-  float GetQ(float default_q) const {
-    return (node_ && node_->GetN() > 0) ? node_->GetQ() : default_q;
+  float GetQ(float default_q, bool logit_q = false) const {
+    return (node_ && node_->GetN() > 0) ?
+      (logit_q ? FastLogit(0.99999994f * node_->GetQ()) : node_->GetQ()) :
```

Comment:
Why such a weird constant? Maybe add comment describing it.

Reply:
I used what @borg323 suggested. I'm pretty sure he picked this since it's the largest 32-bit float less than 1. I can take the final 4 off so it doesn't look as weird without affecting the behavior much since it's about 8.4 for logit(0.9999999) instead of about 8.7.

I'll leave a comment as well.

Reply:
Yes, this is 0x3f7fffff.